packages/next/src/build/utils.ts

```diff
@@ -1391,6 +1391,9 @@ export async function buildAppStaticPaths({
         incrementalCache,
         supportsDynamicHTML: true,
         isRevalidate: false,
+        experimental: {
+          after: false,
```

Comment:
this might yield a confusing error message ("enable `experimental.after` in next.config.js") even if it's enabled, because there's currently no good way to encode "enabled, but not allowed in this context". however, this _should_ trigger a bailout before it gets a chance to print the other message, but i need to double-check that.